packages/account/src/test-utils/launchNode-singular-test.test.ts

```diff
@@ -0,0 +1,20 @@
+import { launchNode } from './launchNode';
+
+/**
+ * The test runner creates a test environment per file,
+ * which we can use to isolate the faulty behavior.
+ */
+/**
+ * @group node
+ */
+describe('launchNode-singular-test', () => {
+  let killedNodeUrl = '';
+  afterAll(async () => {
+    await expect(fetch(killedNodeUrl)).rejects.toThrow('fetch failed');
```

Comment:
You'll need to update this since the url is empty.
```suggestion
  let killedNodeUrl = '/deadnode';
  afterAll(async () => {
    await expect(fetch(killedNodeUrl)).rejects.toThrow('Failed to parse URL from deadnode');
```

Reply:
The `killedNodeUrl` value is updated in the test below and `fetch failed` is the error message thrown when a connection cannot be established.

Reply:
Okay, I just saw the failing tests :smile: I'm investigating.